analysers/analyser_merge_charging_station_FR.py

```diff
@@ -73,21 +212,142 @@ def __init__(self, config, logger=None):
                     },
                     mapping2={
                         "email": "contact_operateur",
-                        "phone": "telephone_operateur",
+                        "operator:phone": "telephone_operateur",
+                        "operator:email": "contact_operateur",
+                        "start_date": "date_mise_en_service",
                         "capacity": "nbre_pdc",
+                        "charging_station:output": lambda fields: self.socket_output_find_correspondances(
+                            fields["puissance_nominale"]) if fields["puissance_nominale"] else None,
```

Comment:
Make sure socket_output_find_correspondances returns None too for invalid values.
I marked most of them